README.md

```diff
@@ -28,8 +27,9 @@ CodeWars (just a heads up: you're required to answer a couple of super basic JS
 
 ## Quality Rubric
 
+- Writing present tense git commit messages: 25 pts
 - All exercises in the Mostly Awesome Guide are completed and pushed to GitHub: 25 pts
-- At least 5 exercises on CodeWars are completed: 25 pts
+- At least 10 exercises on CodeWars are completed: 25 pts
```

Comment:
The Quality rubric and the main specs are inconsistent. One says complete 5, and the other says complete 10. Which is it? The quality rubric should be linked to non-stretch goals only.